templates/hypershift-addon-downstram-image.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.hypershiftAddonMgrOperator.name }}-downstream-override
+  namespace: {{ .Values.global.namespace }}
+data:
+  downstream: |
+    kind: ImageStream
+    apiVersion: image.openshift.io/v1
+    metadata:
+      name: {{ .Values.hypershiftAddonMgrOperator.name }}-downstream-image-stream
+    spec:
+      lookupPolicy:
+        local: false
+      tags:
+      - name: apiserver-network-proxy
+        annotations:
+          io.openshift.build.commit.id: ba74ca29f6cead6c103a109024adc83d4fe6096f
+          io.openshift.build.commit.ref: ''
+          io.openshift.build.source-location: https://github.com/openshift/apiserver-network-proxy
+        from:
+          kind: DockerImage
+          name: quay.io/stolostron/apiserver-network-proxy@latest
```

Comment:
I think we should read the `name` value from the `values.yam` file?

Reply:
yup, I will do that